frontend/src/component/project/ProjectList/ProjectList.tsx

```diff
@@ -152,9 +173,9 @@ export const ProjectListNew = () => {
             ? `${filteredProjects.length} of ${projects.length}`
             : projects.length;
 
-    if (projects?.length === 1) {
-        return <Navigate to={`/projects/${projects[0].id}`} replace />;
-    }
+    // if (projects?.length === 1) {
+    //     return <Navigate to={`/projects/${projects[0].id}`} replace />;
+    // }
```

Comment:
I think this should probably be reverted?

Reply:
the other PR overrides it /fixes it (this is the buggy version). This functionality has been moved to App